lib/fog/google/requests/storage/put_bucket.rb

```diff
@@ -14,13 +14,15 @@ class Real
         # * response<~Excon::Response>:
         #   * status<~Integer> - 200
         def put_bucket(bucket_name, options = {})
-          if location_constraint = options.delete("LocationConstraint")
-            data =
-<<-DATA
-  <CreateBucketConfiguration>
-    <LocationConstraint>#{location_constraint}</LocationConstraint>
-  </CreateBucketConfiguration>
-DATA
+          location_constraint = options.delete("LocationConstraint")
+          storage_class = options.delete("StorageClass")
+          if location_constraint || storage_class
+            data = '<CreateBucketConfiguration>'
+
+            data += "<LocationConstraint>#{location_constraint}</LocationConstraint>" if location_constraint
+            data += "<StorageClass>#{storage_class}</StorageClass>" if storage_class
+            data += '</CreateBucketConfiguration>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.